### PR TITLE
fix: Segmentation slice range is wrong when nearly orthonormal as well as for segmentation volumes

### PR DESCRIPTION
**Pull request by wayfarer3130**
If a segmentation is created with a much larger range than one of the display sets it applies to, then that display set is shown with large swathes of black, and the scrolling can be wrong because it uses the finer of the two scrolls, which can end up duplicating base images.

This change uses primary volumes, which it determines by those having a volume image loader prefix, for slice range, and only uses secondary volumes when there are no primaries.

### CODE REVIEW

File: packages/core/src/utilities/getTargetVolumeAndSpacingInNormalDir.ts [hunk 1]
```diff
@@ -7,6 +7,12 @@ import getSpacingInNormalDirection from './getSpacingInNormalDirection';
 // One EPSILON part larger multiplier
 const EPSILON_PART = 1 + EPSILON;
 
+// Check if this is a primary volume
+// For now, that means it came from some sort of image loader, but
+// should be specifically designated.
+const isPrimaryVolume = (volume): boolean =>
+  volume.volumeId.indexOf(':') !== -1;
```
Comment (sedghi) [inline packages/core/src/utilities/getTargetVolumeAndSpacingInNormalDir.ts:14]: can we add a method to `volumeLoader` to `get` all loaders and just extract the scope and use it here instead?

File: packages/streaming-image-volume-loader/package.json [hunk 2]
```diff
@@ -2,9 +2,12 @@
...
   "version": "0.15.5",
   "description": "",
-  "main": "dist/umd/index.js",
+  "main": "dist/esm/index.js",
   "types": "dist/esm/index.d.ts",
-  "module": "dist/esm/index.js",
+  "module": "src/index.ts",
+  "exports": {
+    ".": "./dist/umd/index.js"
+  },
```
Comment (sedghi) [inline packages/streaming-image-volume-loader/package.json:10]: can you remove this for now, we do it in the other PR

File: packages/streaming-image-volume-loader/api-extractor.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "extends": "../../api-extractor.json",
   "projectFolder": ".",
-  "mainEntryPointFilePath": "<projectFolder>/dist/cjs/index.d.ts",
+  "mainEntryPointFilePath": "<projectFolder>/dist/esm/index.d.ts",
```
Comment (sedghi) [inline packages/streaming-image-volume-loader/api-extractor.json:4]: other `api-extractor`s use cjs, why this one is only esm, can we do it in another PR?

**Review by sedghi — COMMENTED** [review]
see my comments

File: packages/streaming-image-volume-loader/api-extractor.json (hunk 3 above)
Comment (wayfarer3130) [reply packages/streaming-image-volume-loader/api-extractor.json:4]: Thought I had excluded that

**Review by sedghi — APPROVED** [review]
